homeassistant/components/file/notify.py

```diff
@@ -53,16 +60,32 @@ def send_message(self, message: str = "", **kwargs: Any) -> None:
         """Send a message to a file."""
         file: TextIO
         filepath: str = os.path.join(self.hass.config.config_dir, self.filename)
-        with open(filepath, "a", encoding="utf8") as file:
-            if os.stat(filepath).st_size == 0:
-                title = (
-                    f"{kwargs.get(ATTR_TITLE, ATTR_TITLE_DEFAULT)} notifications (Log"
-                    f" started: {dt_util.utcnow().isoformat()})\n{'-' * 80}\n"
-                )
-                file.write(title)
+        if not self.hass.config.is_allowed_path(filepath):
```

Comment:
Should this not be checked in `async_get_service` instead so we don't load the class if file isn't accessible?

Reply:
Okay this part is moved to another PR (see docs PR)

Reply:
Checks moved to the entry setup for both notify and sensor.